pkgs/top-level/all-packages.nix

```
@@ -39116,6 +39116,8 @@ with pkgs;
 
   zoneminder = callPackage ../servers/zoneminder { };
 
+  zrok-bin = callPackage ../tools/networking/zrok-bin { };
```

```suggestion
  zrok = callPackage ../tools/networking/zrok { };
```
please set meta.sourceProvenance to reflect this